reproducibility_FAQs.md

```diff
@@ -0,0 +1,253 @@
+
+## Table of Contents
+
+- [General FAQs](#general-faqs)
+  - [How do I submit a package?](#how-do-i-submit-a-package)
+  - [How long does the reproducibility verification process take?](#how-long-does-the-reproducibility-verification-process-take)
+  - [How should I organize my package?](#how-should-i-organize-my-package)
+  - [What does the reproducibility report contain?](#what-does-the-reproducibility-report-contain)
+  - [What are the most common reasons reproducibility checks fail?](#what-are-the-most-common-reasons-reproducibility-checks-fail)
+  - [What if my package uses Excel?](#what-if-my-package-uses-excel)
+
+- [Data-Related Questions](#data-related-questions)
+  - [What is original data?](#what-is-original-data)
+  - [Can you do a reproducibility verification if the research relies on confidential data?](#can-you-do-a-reproducibility-verification-if-the-research-relies-on-confidential-data)
+  - [How do I document the datasets used in the reproducibility package?](#how-do-i-document-the-datasets-used-in-the-reproducibility-package)
+
+- [Code-Related Questions](#code-related-questions)
+  - [How should I organize my code?](#how-should-i-organize-my-code)
+  - [What is a main script? How do I create one?](#what-is-a-main-script-how-do-i-create-one)
+  - [How do I check if my code is stable?](#how-do-i-check-if-my-code-is-stable)
+  - [How do I set up a reproducible environment?](#how-do-i-set-up-a-reproducible-environment)
+  - [Do you check if the code is correct?](#do-you-check-if-the-code-is-correct)
+
+- [Documentation-Related Questions](#documentation-related-questions)
+  - [What is a README and what should it include?](#what-is-a-readme-and-what-should-it-include)
+  - [What is a Data Availability Statement?](#what-is-a-data-availability-statement)
+
+
+## General FAQs
+
+### How do I submit a package?
+Authors can request a reproducibility verification through [this submission form](https://survey.wb.surveycto.com/collect/prwp_reproducibility?caseid=).    The package must include all components listed in this [checklist](https://github.com/worldbank/wb-reproducible-research-repository/blob/main/reproducibility_package_checklist.md).
+
+### How long does the reproducibility verification process take?
+
+Most packages are reviewed within 2 weeks when only minor or no corrections are needed. The total time depends on the complexity of the analysis and the completeness of the submission. Packages requiring substantial clarification or additional information from authors may take longer.
+
+
+### How should I organize my package?
+
+It is  recommended to organize your reproducibility package using a clear and consistent folder structure.  A standard structure includes:
+-   `Data/`:    Include only  **original input data**  as described in the README. Do  not  include intermediate or processed files.
```

Comment:
we sometimes do want intermediate data though right? if we can't run the source data directly?